### DISCUSSION TITLE
process.env.DOCUSAURUS_CURRENT_LOCALE is undefined in serve

### DISCUSSION THREAD

**Discussion by laojun** [facebook/docusaurus#8848] [Q&A]
Hi:
First of all, Thanks for the v2.4。

I use `process.env.DOCUSAURUS_CURRENT_LOCALE`  to solve my problem in https://github.com/facebook/docusaurus/discussions/8745。

Here is my docusaurus.config.js
```
baseUrl: `/${process.env.DOCUSAURUS_CURRENT_LOCALE}/`
```

Seems right when i run build
![image](https://user-images.githubusercontent.com/15364706/228841951-1cc283a1-473e-4196-b920-9637f9b14eb9.png)

But error when i run server
![image](https://user-images.githubusercontent.com/15364706/228842944-d4ae8c65-7c3e-4201-8f53-fb064f14b5e3.png)

How can i make it right when i run server?

Thanks

**Comment by Josh-Cena** [comment]
Thanks @laojun this is a bug. I've opened https://github.com/facebook/docusaurus/issues/8849.
